packages/composables/src/useListing/useListing.ts

```diff
@@ -36,6 +36,17 @@ function merge<T extends { [key in keyof T]: unknown }>(
 
 export type ListingType = "productSearchListing" | "categoryListing";
 
+export type ShortcutFilterParam = {
+  code:
+    | "manufacturer"
+    | "min-price"
+    | "max-price"
+    | "rating"
+    | "shipping-free"
+    | "properties";
```

Comment:
what about custom filters?
This probably should be a `keyof ProductListingCriteria` from the Schema

Reply:
Would currently not work with setCurrentFilters but was also never working :thinking: